rfcs/README.md

```diff
@@ -0,0 +1,62 @@
+- A new Gestalt utility component (such as [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) or [useReducedMotion](https://gestalt.pinterest.systems/usereducedmotion)).
+- A new feature that creates new API surface area (such as [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) and `dangerouslyDisableOnNavigation`).
+- The introduction of new idiomatic usage, conventions, or patterns (such as [Gestalt design tokens](https://gestalt.pinterest.systems/design_tokens), [boint units](https://gestalt.pinterest.systems/faq#Component-usage), or subcomponents modularity patterns like [Table](https://gestalt.netlify.app/table#Props) or [Dropdown](https://gestalt.netlify.app/dropdown) subcomponents).
+- Technology migrations (such as the [migration to Next.js](https://github.com/pinterest/gestalt/pull/1642))
+
+Some changes do not require an RFC:
+
+- Rephrasing, reorganizing or refactoring
+- New general components
+- Additions that strictly improve objective, numerical quality
+  criteria (speedup, better browser support)
+
+If you submit a pull request to implement a new feature without going
+through the RFC process, it may be closed with a polite request to
+submit an RFC first.
+
+## How to submit an RFC
+
+1. Create a directory inside this `rfcs` directory. The directory name should begin with the year and include a meaningful description, such as `rfcs/2022-onInteraction-new-feature`.
+2. Copy the [template.md](https://github.com/pinterest/gestalt/tree/master/rfcs/template.md) file from this rfcs directory into your newly created subdirectory (such as `rfcs/2022-onInteraction-new-feature/README.md`). Be sure to name your file README.md so it is easily viewable in the GitHub interface.
+3. If you want to include images in your RFC, place them in the same directory as the README.md.
+4. Fill in the RFC. Please fill in every section in the template with as much detail as possible.
+5. Submit a pull request to this repo with all of your files.
+6. You will receive feedback both from the Gestalt team. You should be prepared to update your RFC based on this feedback. The goal is to build consensus on the best way to implement the suggested change.
+7. When all feedback has been incorporated, the Gestalt team will determine whether or not to accept the RFC. RFCs that are accepted will be merged directly into this repo; RFCs that are not accepted will have their pull requests closed without merging.
+
+## The RFC life-cycle
+
+Once an RFC is merged into this repo, then the authors may implement it and submit a pull request. Note that the implementation still needs to be reviewed separate from the RFC, so you should expect more feedback and iteration.
+
+If the RFC authors choose not to implement the RFC, then the RFC will be removed from the RFC repository.
```

Comment:
We probably shouldn't delete anything — even discarded ideas are good history. Perhaps we need an "archive" directory or something similar?

Reply:
DONE